.ruby-version

```diff
@@ -1 +1 @@
-2.3.0
+2.3.4
```

Comment:
I am quite certain that this file is not used during compilation, It is supposed to mean more like a hint for ruby managers such as rvm and rbenv =)